catalog/geoscience-aus-cat/projects_geoscience-aus-cat_assets_ga_s2ls_intertidal_cyear_3.jsonnet

```diff
@@ -0,0 +1,135 @@
+local id = 'projects/geoscience-aus-cat/assets/ga_s2ls_intertidal_cyear_3';
```

Comment:
the thumbnali is very similar to the one for https://developers.google.com/earth-engine/datasets/catalog/projects_geoscience-aus-cat_assets_NIDEM

Does GA plan to mark NIDEM as deprecated by this dataset? If yes, we might want to hide NIDEM from the main list at https://developers.google.com/earth-engine/datasets/publisher/geoscience-aus-cat . If no, is it possible to use a slightly different thumbnail?